app/src/renderer/components/common/TmPage.vue

```diff
@@ -80,7 +81,7 @@ export default {
       default: ``
     },
     error: {
-      type: Object,
+      type: Object | Error,
```

Comment:
what is this `|` ?

Reply:
Object or Error, I received sometimes this warning by Vue

Reply:
Didn't we decide to not allow different prop types?

Reply:
I also vote for not having `Error` here

Reply:
Ok, I'll drop this and when the warnings appear I'll open an issue so we can handle the errors properly